CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- On Windows, fix freezes when performing certain actions after a window resize has been triggered.
```

Comment:
I'd appreciate if this also mentioned the reintroduction of the visual bug. (There should also be a newline between the heading and the list.)